dts/msc-sm2-imx6x.dtsi

```diff
@@ -342,7 +342,7 @@
 		};
```

Comment:
Do you remember what error and why the node name is not correct?

Reply:
Perhaps, but I'm gonna need more than just a closing bracket and a semicolon.

Reply:
I meant the reason why this commit is made